src/backend/distributed/transaction/citus_dist_stat_activity.c

```diff
@@ -253,6 +253,8 @@ static TimestampTz ParseTimestampTzFieldFromHeapTuple(HeapTuple tuple, TupleDesc
 													  int colIndex);
 static TransactionId ParseXIDFieldFromHeapTuple(HeapTuple tuple, TupleDesc tupdesc, int
 												colIndex);
+static void ReplaceInitiatorNodeIdentifier(int initiator_node_identifier,
```

Comment:
Super minor coding convention notes:

We usually prefer to declare & define the function right after the first function that uses them. In this case, it'd be good to move the declaration right after `ParseCitusDistStat `. As far as I can see, the definition is already on the correct place.

Reply:
putting definition right after the usage makes it easy to follow the flow of the code, but what benefit does having declaration in the same order?

(I have updated the declaration too)

Reply:
I think you can trace the related functions a bit easier